src/FiberTraces.cc

```diff
@@ -2898,67 +2900,67 @@ template PTR(pfsDRPStella::FiberTrace< double, unsigned short, float >) pfsDRPSt
                                                                                                  double,
                                                                                                  size_t );
 
-template bool pfsDRPStella::math::addFiberTraceToCcdArray( pfsDRPStella::FiberTrace< float, unsigned short, float > const&,
+template void pfsDRPStella::math::addFiberTraceToCcdArray( pfsDRPStella::FiberTrace< float, unsigned short, float > const&,
```

Comment:
Hmmm... Could you give me an example please? Couldn't find one on the internet doing a quick search...